### PR TITLE
setup project structure

### PR DESCRIPTION
**Pull request by AhmedMohsen600**
## What does this PR do
In this PR me and my coding partner check the error about the feature and make the changes.
## Description of the task to be completed
- [x] Setup project structure

## How should it be tested
Clone the repository
```
git clone https://github.com/AhmedMohsen600/Leaderboard.git
```
Navigate to the folder
```
cd Leaderboard
```
git checkout feature
```
#### Thanks for reviewing my PR 😊

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @AhmedMohsen600,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [x] Some linters checks are not passed. Please make sure that all checks are passing and green.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      <h1>Leaderboard</h1>
+      <div class="group">
+        <div class="score-holder">
+          <div class="recent-score">
+            <h3></h3>
+            <button class="refresh">Refresh</button>
+          </div>
+          <div class="score-table"></div>
+        </div>
+        <div class="form-group">
```
Comment (christianbiring1) [inline dist/index.html]: - [ ] As stated in the project requirement, Kindly consider adding some markup to your page that will be a little bit similar to the provided wireframe. `some hard-coded scores and name` :smile: 
YOur design :point_down: 
image

Template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @AhmedMohsen600 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Webpack is set correcttl to bundle the Js
- [ ] Correct git-flow is used :+1: 
- [ ] Descriptive PR `title` and `summary`

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  width: 100%;
+}
+
+input {
+  padding: 10px;
+}
```
Comment (ShahierNashaat) [inline src/style.css:92]: - [ ] [OPTIONAL] Kindly notice that you should add these lines of CSS code to make the button at the right of the form as it is in the wireframe.

```css
width: fit-content;
align-self: end;
```

**Review by ShahierNashaat — APPROVED** [review]
Hi @AhmedMohsen600, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**